Makefile

```diff
@@ -4,6 +4,7 @@
 
 GIT_COMMIT := $(shell git rev-parse --verify HEAD)
 GIT_VERSION := $(shell git describe --dirty)
+GO_VERSION := $(shell go list -f {{.GoVersion}} -m)
```

Comment:
this means we are using the locally installed go version as the version to use in the docker container.

we then need to make sure the building host has the appropriate go version.

is it possible we set the golang version here directly?

Reply:
This actually grabs the go version defined in the go.mod rather than the one installed on the host. But you are correct, the host must have go installed in order to run the command.